tests/unit/test_report_builder.py

```diff
@@ -0,0 +1,68 @@
+"""
+test_jinja_builder.py
+Date:  09/09/2020
+Author: Laurentiu Mandru
+Email: mclaurentiu79@gmail.com
+"""
+
+import pathlib
+import shutil
+
+import pytest
+from eds4jinja2.builders.report_builder import ReportBuilder
+from bs4 import BeautifulSoup
+
+
+@pytest.fixture(scope="function")
+def sample_data_path() -> str:
+    return pathlib.Path(__file__).parent.parent / "test_data/templates_test"
+
+
+# @pytest.fixture(autouse=True)
+# def setup_teardown(sample_data_path):
+#     yield
+#     path = pathlib.Path(sample_data_path) / "output"
+#     shutil.rmtree(path)
+
+
+before_fired = False
+after_fired = False
+
+
+def before_listener():
+    global before_fired
```

Comment:
let's get rid of global variables and pass them as parameters or shared context.